attestation-service/verifier/Cargo.toml

```diff
@@ -11,7 +11,7 @@ sgx-verifier = [ "scroll", "sgx-dcap-quoteverify-rs" ]
 az-snp-vtpm-verifier = [ "az-snp-vtpm", "sev", "snp-verifier" ]
 snp-verifier = [ "asn1-rs", "openssl", "sev", "x509-parser" ]
 csv-verifier = [ "openssl", "csv-rs", "codicon" ]
-cca-verifier = [ "cbor-diag", "veraison-apiclient" ]
+cca-verifier = [ "veraison-apiclient" ]
```

Comment:
not related to this PR. Could you also add `ear` dependency as optional under cca-verifier feature?

Reply:
fixed.